cmd/podman/runlabel.go

```diff
@@ -154,7 +152,21 @@ func runlabelCmd(c *cli.Context) error {
 	}
 
 	if pull {
-		newImage, err = runtime.ImageRuntime().New(ctx, runlabelImage, rtc.SignaturePolicyPath, "", stdOut, nil, image.SigningOptions{}, false, false)
+		var registryCreds *types.DockerAuthConfig
+		if c.IsSet("creds") {
+			creds, err := util.ParseRegistryCreds(c.String("creds"))
+			if err != nil {
+				return err
+			}
+			registryCreds = creds
+		}
+		dockerRegistryOptions := image.DockerRegistryOptions{
+			DockerRegistryCreds:         registryCreds,
+			DockerCertPath:              c.String("cert-dir"),
+			DockerInsecureSkipTLSVerify: !c.BoolT("tls-verify"),
+		}
+
+		newImage, err = runtime.ImageRuntime().New(ctx, runlabelImage, c.String("signature-policy"), c.String("authfile"), stdOut, &dockerRegistryOptions, image.SigningOptions{}, false, false)
```

Comment:
Probably ought to merge this before the authFile change @QiWang19 is working on.  Pretty sure there'll be a merge dance between the two.